RadarSDK/RadarVerifyServer.swift

```diff
@@ -0,0 +1,174 @@
+//
+//  RadarVerifyServer.swift
+//  RadarSDK
+//
+//  Created by Nick Patrick on 11/11/24.
+//  Copyright © 2024 Radar Labs, Inc. All rights reserved.
+//
+
+import Foundation
+import UserNotifications
+
+#if canImport(Telegraph)
+import Telegraph
+
+@objc(RadarVerifyServer) class RadarVerifyServer: NSObject, CLLocationManagerDelegate {
+    @MainActor @objc static let sharedInstance = RadarVerifyServer()
```

Comment:
The modifier `@MainActor` forces the operations here to be done on the main thread. Is there a specific need for us to do all these non UI tasks on the main thread? This may be especially problematic if the app is running in the background and iOS imposes limitations on the app's main thread utilization (watchdog).